pyriemann/transferlearning.py

```diff
@@ -0,0 +1,147 @@
+from .classification import MDM
+import numpy as np
+from joblib import Parallel, delayed
+
+from .utils.mean import mean_covariance
+from .utils.geodesic import geodesic
+
+class MDWM (MDM):
+    """Classification by Minimum Distance to Weighted Mean.
+
+    Classification by nearest centroid. For each of the given classes, a
+    centroid is estimated, according to the chosen metric, as a weighted mean
+    of point (i.e. covariance matrices) from the source domain, combined with
+    the class centroid of the target domain.
+    For classification, a given new point is attibuted to the class whose
+    centroid is the nearest according to the chosen metric.
+
+    Parameters
+    ----------
+    metric : string | dict (default: 'riemann')
+        The type of metric used for centroid and distance estimation.
+        see `mean_covariance` for the list of supported metric.
+        the metric could be a dict with two keys, `mean` and `distance` in
+        order to pass different metric for the centroid estimation and the
+        distance estimation. Typical usecase is to pass 'logeuclid' metric for
+        the mean in order to boost the computional speed and 'riemann' for the
+        distance in order to keep the good sensitivity for the classification.
+    L : float, (default: 0)
+        Transfer coefficient in [0,1]. This parameter controls the trade-off
+        between source and target data.
```

Comment:
This description should be completed: 
```suggestion
    L : float, (default: 0)
        Transfer coefficient in [0,1], controlling the trade-off between 
        source and target data. At 0, there is no transfer, only the data
        acquired from the source are used.  At 1, this is a calibration-free 
        system as no data are required from the source.
```
I have no idea to replace `L`. What about `lambda`?

Reply:
`lambda` seems good!

Reply:
`lambda` generates an error, because it is a reserved keyword...
What about `transfer_tradeoff` ?

Reply:
good to me!